Content.Shared/_EE/Flight/Events.cs

```diff
@@ -0,0 +1,24 @@
+using Robust.Shared.Serialization;
+using Content.Shared.DoAfter;
+
+namespace Content.Shared._EE.Flight.Events;
+
+[Serializable, NetSerializable]
+public sealed partial class DashDoAfterEvent : SimpleDoAfterEvent { }
+
+[Serializable, NetSerializable]
+public sealed partial class FlightDoAfterEvent : SimpleDoAfterEvent { }
+
+[Serializable, NetSerializable]
+public sealed class FlightEvent : EntityEventArgs
+{
+    public NetEntity Uid { get; }
+    public bool IsFlying { get; }
+    public bool IsAnimated { get; }
+    public FlightEvent(NetEntity uid, bool isFlying, bool isAnimated)
+    {
+        Uid = uid;
+        IsFlying = isFlying;
+        IsAnimated = isAnimated;
+    }
+}
```

Comment:
Would be nicer like this I believe
```suggestion
[Serializable, NetSerializable]
public sealed partial class FlightDoAfterEvent : SimpleDoAfterEvent;

[Serializable, NetSerializable]
public sealed class FlightEvent(NetEntity uid, bool isFlying, bool isAnimated) : EntityEventArgs
{
    public NetEntity Uid { get; } = uid;
    public bool IsFlying { get; } = isFlying;
    public bool IsAnimated { get; } = isAnimated;
}
```